### DISCUSSION TITLE
Running ignition

### DISCUSSION THREAD

**Discussion by till** [flatcar/Flatcar#1390] [Q&A]
I am building Docker images with Flatcar and was wondering if it's possible to "manually" (re-)run ignition like in the boot process to test additions to images?

Thanks in advance!

**Comment by tormath1** [comment]
Hello, maybe you can have a look to this: https://www.flatcar.org/docs/latest/provisioning/ignition/boot-process/#reprovisioning

There is a new tool called `ignition-apply` that allows to run Ignition from containers too: https://coreos.github.io/ignition/release-notes/#ignition-2140-12-may-2022 (never tried)

**Reply by pothos** [reply]
There is also `flatcar-reset --preview-delete` and `--preview-keep` which can be used instead of the `find` invocation for testing regexes

**Comment by PackElend** [comment]
That is exactly what I need to do bit where does it pull the ignition file from?

Any chance to refer to a adapted ignition file ?

**Reply by PackElend** [reply]
is being answered in https://github.com/flatcar/Flatcar/issues/1984
